packages/utils/src/helpers/domain.ts

```diff
@@ -41,6 +41,29 @@ export const chainIdToDomainMapping: Map<number, number> = new Map([
   [13338, 13338],
 ]);
 
+export const mainnetDomains: number[] = [
```

Comment:
I feel like this list has existed in a few forms -- can we make sure everything is using this? and since its a `const` it should really be `MAINNET_DOMAINS`